routes/stats.js

```diff
@@ -1,12 +1,245 @@
 
 module.exports = router;
+
+/**
+ * @swagger
+ * components:
+ *   schemas:
+ *     GlobalStatsResponse:
+ *       type: object
+ *       properties:
+ *         success:
+ *           type: boolean
+ *           example: true
+ *         status:
+ *           type: string
+ *           example: success
+ *         statusCode:
+ *           type: integer
+ *           example: 200
+ *         message:
+ *           type: string
+ *           example: Success
+ *         data:
+ *           type: object
+ *           properties:
+ *             totalUsers:
+ *               type: string
+ *               example: "78"
+ *             totalComments:
+ *               type: string
+ *               example: "2"
+ *             totalProjects:
+ *               type: string
+ *               example: "26"
+ *             averageLifespan:
+ *               type: string
+ *               example: "7.0000000000000000"
+ *             mostCommonCause:
+ *               type: string
+ *               example: "Bored"
+ *             funeralsToday:
+ *               type: string
+ *               example: "0"
+ *             totalVotes:
+ *               type: string
+ *               example: "2"
+ *             votesToday:
+ *               type: string
+ *               example: "0"
+ *             averageEulogyLength:
+ *               type: string
+ *               example: "137.8846153846153846"
+ *             rageQuitRate:
+ *               type: string
+ *               example: "23.0769230769230769"
+ *             topBurialDay:
+ *               type: string
+ *               example: "Wednesday"
+ *             topBurialMonth:
+ *               type: string
+ *               example: "January"
+ *             mostVotedProject:
+ *               type: object
+ *               properties:
+ *                 id:
+ *                   type: string
+ *                   format: uuid
+ *                   example: "aa177205-6aec-4f7c-934c-ebcbfdcb9df9"
+ *                 name:
+ *                   type: string
+ *                   example: "Book Reading List"
+ *                 votes:
+ *                   type: integer
+ *                   example: 1
+ *     UserStatsResponse:
+ *       type: object
+ *       properties:
+ *         success:
+ *           type: boolean
+ *           example: true
+ *         status:
+ *           type: string
+ *           example: success
+ *         statusCode:
+ *           type: integer
+ *           example: 200
+ *         message:
+ *           type: string
+ *           example: Success
+ *         data:
+ *           type: object
+ *           properties:
+ *             totalComments:
+ *               type: string
+ *               example: "1"
+ *             totalProjects:
+ *               type: string
+ *               example: "4"
+ *             averageLifespan:
+ *               type: string
+ *               example: "23.7500000000000000"
```

Comment:
Probably me missing something, but otherwise:

```suggestion
 *             totalComments:
 *               type: integer
 *               example: 1
 *             totalProjects:
 *               type: integer
 *               example: 4
 *             averageLifespan:
 *               type: number
 *               format: float
 *               example: 23.7500000000000000
```